changelogs/unreleased/implement-getting-network-statistic-per-thread.md

```diff
@@ -0,0 +1,7 @@
+## feature/core
+
+ * Getting network statistics per thread was implemented. User has
+   possibility to run several iproto threads, but at the moment he
+   can get only general statistics, which combines statistics for
+   all threads. Now `threads` metric was added, it contains statistics
+   per thread.
```

Comment:
Haven't looked into the code, but I can say right away that I don't like the output format, because
 - it's impossible to say which thread a certain counter belongs to;
 - it's not even possible to match SENT and RECEIVED counters of the same thread.

IMO it should be done vice-versa: the output should be organized as a map: thread -> name/port + stat counters. Maybe, the map should be returned by box.stat.net(). Maybe, there should be a separate call (say box.stat.net.thread()). Not sure.